components/SingleEntry/Entry.js

```diff
@@ -90,22 +104,37 @@ export default function Entry({ id }) {
           </Container>
         )}
       </AnimatePresence>
-      <ToolsContainer>
-        {/* <DeleteButton as="a" onClick={() => router.back()}>
-          {showSentence ? (
-            <FiArrowLeft />
-          ) : (
-            <FiArrowLeft style={{ color: "grey", opacity: "0.5" }} />
-          )}
-        </DeleteButton> */}
-        <DeleteButton as="a" onClick={() => handleShowSentence()}>
-          {showSentence ? (
-            <FaRegEyeSlash />
+      {session && (
+        <ToolsContainer>
+          {deletingId === entry._id ? (
+            <DeleteButton
+              style={{ color: "red" }}
+              as="a"
```

Comment:
remove the inline styling here as well. Why do you want to display the Button element as an "a" tag?